front/components/skills/SkillsTable.tsx

```diff
@@ -28,125 +29,160 @@ type RowData = {
+          <div>
+            <SkillAvatar />
+          </div>
+          <div className="flex min-w-0 grow flex-col">
+            <div className="heading-sm overflow-hidden truncate text-foreground dark:text-foreground-night">
+              {info.getValue()}
+            </div>
+            <div className="overflow-hidden truncate text-sm text-muted-foreground dark:text-muted-foreground-night">
+              {info.row.original.description}
+            </div>
+          </div>
+        </div>
+      </DataTable.CellContent>
+    );
+  },
+  meta: {
+    className: "w-40 @lg:w-full",
+  },
+};
+
+const editorsColumn = {
+  header: "Editors",
+  accessorKey: "editors",
+  cell: (info: CellContext<RowData, UserType[]>) => {
+    const editors = info.getValue();
+    const items = editors
+      ? editors.map((editor) => ({
+          name: editor.fullName,
+          visual: editor.image,
+        }))
+      : // Only dust managed skills should have no editors
+        [
+          {
+            name: "Dust",
+            visual: DUST_AVATAR_URL,
+          },
+        ];
+    return <DataTable.CellContent avatarStack={{ items, nbVisibleItems: 4 }} />;
+  },
+  meta: {
+    className: "hidden @sm:w-32 @sm:table-cell",
+  },
+};
+
+const usedByColumn = (onAgentClick: (agentId: string) => void) => ({
+  header: "Used by",
+  accessorKey: "usage",
+  cell: (info: CellContext<RowData, AgentsUsageType>) => (
+    <DataTable.CellContent>
+      <UsedByButton usage={info.getValue()} onItemClick={onAgentClick} />
+    </DataTable.CellContent>
+  ),
+  meta: {
+    className: "hidden @sm:w-24 @sm:table-cell",
+  },
+});
+
+const lastEditedColumn = {
+  header: "Last Edited",
+  accessorKey: "updatedAt",
+  cell: (info: CellContext<RowData, number | null>) => {
+    const value = info.getValue();
+    return (
+      <DataTable.BasicCellContent
+        tooltip={value ? formatTimestampToFriendlyDate(value, "long") : ""}
+        label={value ? formatTimestampToFriendlyDate(value, "compact") : ""}
+      />
+    );
+  },
+  meta: { className: "hidden @sm:w-32 @sm:table-cell" },
+};
+
+const suggestionDateColumn = {
+  header: "Suggestion date",
+  accessorKey: "createdAt",
+  cell: (info: CellContext<RowData, number | null>) => {
+    const value = info.getValue();
+    return (
+      <DataTable.BasicCellContent
+        tooltip={value ? formatTimestampToFriendlyDate(value, "long") : ""}
+        label={value ? formatTimestampToFriendlyDate(value, "compact") : ""}
+      />
+    );
+  },
+  meta: { className: "hidden @sm:w-32 @sm:table-cell" },
+};
+
+const menuColumn = {
+  header: "",
+  accessorKey: "menuItems",
+  cell: (info: CellContext<RowData, MenuItem[]>) => {
+    return <DataTable.MoreButton menuItems={info.getValue()} />;
+  },
+  meta: {
+    className: "w-14",
+  },
+};
+
+const getTableColumns = (
+  onAgentClick: (agentId: string) => void,
+  variant: "default" | "suggested"
```

Comment:
Seen IRL, pass the activeTab to SkillsTab rather than adding complexity with a new variable "variant"